tutorials/2021-06-17-writing-a-nep17-token-in-python/index.md

```diff
@@ -61,6 +48,15 @@ As can be seen in the native assets contracts linked in the previous section, fu
 
 In this section, we'll give a brief overview of the methods we're going to implement later. Right after, we'll showcase the full code of our token. We'll then proceed to cover the code bit by bit throughout the rest of the tutorial.
 
+## 4. Testing a Neo smart-contract
+
+There are currently 2 ways to test a smart-contract on the Neo platform. Both depend on the [Neo Blockchain Toolkit](https://github.com/neo-project/neo-blockchain-toolkit). We recommend you install it using their VS Code extension.
```

Comment:
This is not `technically` correct and the copy implies that developers must use vscode.